src/Data/Parity/Base.agda

```diff
@@ -0,0 +1,57 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Parity
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Parity.Base where
+
+open import Data.Sign.Base using (Sign; +; -)
+
+------------------------------------------------------------------------
+-- Definition
+
+data Parity : Set where
+  eve : Parity
```

Comment:
Good question. 
You'll be surprised to learn I did agonise over this. 
Pro my choice: two equal length names; no need to rename to avoid clash with existing uses of `even` in `Data.Nat.*` (but what about `odd`, then?)
Contra: evidently idiotic/quixotic
Either way, I also felt the need *immediately* to introduce the pattern synonyms for the sake of the algebraic structure.

Reply:
Yup I also considered suggesting that. I would be happy with that :+1:

Reply:
I'm a -1 on `0ℙ` and `1ℙ`. I don't find them insightful names at all.

Reply:
@Taneb Hmmm: `0 is even parity` `1 is odd parity`?

Reply:
I have gone with `0ℙ` and `1ℙ`.